src/api/questions.test.js

```diff
@@ -556,18 +553,28 @@ describe('Questions API', () => {
     })
 
     test('fails for course staff of different course', async () => {
+      const reqAdmin = await requestAsUser(app, 'admin')
```

Comment:
Can you explain the addition of these lines?

Reply:
Oh, I added those in to change the `questionFeedback` attribute of the first test course to be false, before testing whether a different course's course staff/a student could answer a question, though I guess this is unneeded for the failure test cases, as the route is the same anyways.